update_application.sh

```diff
@@ -26,7 +26,8 @@ else
 fi
 
 # Install dependencies
-cp $REMOTE_SCRIPT_PATH/conf/secrets.js ./
+cp $REMOTE_SCRIPT_PATH/conf/secrets.js ./conf/
+cp $REMOTE_SCRIPT_PATH/conf/env.js ./conf/env.js
```

Comment:
This is ok for now, but I'm wondering if it's useful to have a script flag like  --env prod or something like that. I don't like the idea of blindly copying whatever env.js you have in your repo at the time. Seems dangerous. Feels like it would be better to be explicit. Also might suggest a similar setup for the secrets.js. --secrets /path/to/secrets.js